test/test_outages.jl

```diff
@@ -39,14 +39,14 @@
         ),
     ) == 1
     @test length(
-        get_supplemental_attributes(
+        list_supplemental_attributes(
```

Comment:
```suggestion
        get_supplemental_attributes(
```

Reply:
Yeah, I changed all of these this morning and have been waiting for the CostCurve fix.